tools/clang/test/HLSLFileCheck/d3dreflect/rayquery-reflection.hlsl

```diff
@@ -0,0 +1,14 @@
+// RUN: %dxilver 1.9 | %dxc -T lib_6_9 -validator-version 1.9 %s | FileCheck -check-prefix=NOD3D %s
+// RUN: %dxilver 1.9 | %dxc -T lib_6_9 -validator-version 1.9 %s | %D3DReflect %s | FileCheck -check-prefix=D3D %s
+
+// NOD3D: ; RaytracingPipelineConfig1 rpc = { MaxTraceRecursionDepth = 32, Flags = RAYTRACING_PIPELINE_FLAG_SKIP_TRIANGLES | RAYTRACING_PIPELINE_FLAG_ALLOW_OPACITY_MICROMAPS };
+// D3D: MinShaderTarget: 0x10069
+RaytracingPipelineConfig1 rpc = { 32, RAYTRACING_PIPELINE_FLAG_SKIP_TRIANGLES | RAYTRACING_PIPELINE_FLAG_ALLOW_OPACITY_MICROMAPS };
+
+RaytracingAccelerationStructure RTAS;
+// DXR entry point to ensure RDAT flags match during validation.
```

Comment:
This isn't a DXR entry point.

It wouldn't be checked to match with the subobject even if it was, because how the subobject is used depends on runtime state.  So the D3D12 Runtime will validate this kind of consistency, at least as much as it can.  Since we don't expose the use of `RAY_FLAG_FORCE_OMM_2_STATE` in a DXR entry point to the runtime, it can't actually validate this interaction.

Ultimately, we don't need an entry point in this test, as it doesn't interact with the subobject and the flag.